package.json

```diff
@@ -49,5 +51,11 @@
     "Nate Rajlich <nathan@tootallnate.net> (https://github.com/tootallnate)",
     "Sean McArthur <sean.monstar@gmail.com> (https://github.com/seanmonstar)",
     "Fanie Oosthuysen <fanie.oosthuysen@gmail.com> (https://github.com/weareu)"
-  ]
+  ],
+  "binary": {
+    "module_name": "bcrypt_lib",
+    "module_path": "./lib/binding/",
+    "host": "https://github.com",
+    "remote_path": "/ksmyth/node.bcrypt.js/releases/download/v{version}/"
```

Comment:
Great, I have created the appveyor account. Could you update this url to reference `ncb000gt` vs `ksmyth` and I will merge. I could do this with some git gymnastics but figured would be easiest if done here and PR merged per normal methods.